service_manager_platform_interface/lib/src/implementations/method_channel_service_manager.dart

```diff
@@ -4,7 +4,8 @@ import '../../service_manager_platform_interface.dart';
 
 class MethodChannelServiceManager extends ServiceManagerPlatform {
 
-  MethodChannel methodChannel = MethodChannel('flutter.baseflow.com/service_manager');
+  MethodChannel methodChannel = MethodChannel('flutter.baseflow.com/service_manager/methods');
+  EventChannel eventChannel = EventChannel('flutter.baseflow.com/service_manager/events');
```

Comment:
This eventChannel is only for BluetoothStates, so I would rename it to bluetoothStateChannel.
```suggestion
  EventChannel bluetoothStateChannel = EventChannel('flutter.baseflow.com/service_manager/events/bluetooth');
```